docs/development/local_setup.md

```diff
@@ -320,36 +333,41 @@ $ vagrant destroy --force
 ==> core-01: Forcing shutdown of VM...
 ==> core-01: Destroying VM and associated drives...
 
-$ cd $GOPATH/src/github.com/gardener/gardener
+$ cd ..
 ```
 
-#### Start the Gardener Local Provider
-
-The Seed cluster provides the possibility to create Shoot clusters on several cloud provider. The Gardener Provider implements a common interface to all supported cloud providers. Here, the corresponding Gardener Provider for Local is used.
+#### Create, access and delete a Shoot Cluster
 
-By executing
+Now, you can create a Shoot cluster by running
 
 ```bash
-$ make start-local
-2018/02/14 10:53:34 Listening on :3777
-2018/02/14 10:53:34 Vagrant directory /Users/foo/go/src/github.com/gardener/gardener/vagrant
-2018/02/14 10:53:34 user-data path /Users/foo/git/go/src/github.com/gardener/gardener/dev/user-data
+$ kubectl apply -f dev/90-shoot-local.yaml
+shoot "local" created
 ```
 
-the Gardener Local Provider is started.
+Wait until the 2 secrets `osc-result-cloud-config-local-*` appear in the Shoot cluster namespace and then copy `cloud_config` from the secret `osc-result-cloud-config-local-xxxxx-downloader` to the local file `dev/user-data`. This file is used to pass the downloader configuration to the Vagrant machine, which triggers the mechanism that configures the machine properly and causes it to join the Shoot cluster as a node. 
```

Comment:
Could it be possible to add this as a target in the make file like `setup-local-worker`? Something like:
`hack/dev-download-secret`
```
while [ -z "$DOWLOADER_SECRET" ]; do
         DOWNLOADER_SECRET=$(kubectl get secrets -n shoot--dev--local | grep osc-result-cloud-config-local | grep downloader | cut -d' '  -f )
done
kubectl get secret $DOWNLOADER_SECRET -n shoot--dev--local -o jsonpath="{.data.cloud_config}" | base64 -d > dev/user-data
```
and add this target to the `Makefile`:
```
setup-local-worker`: 
         ./hack/dev-dowload_secret
```

Reply:
Please let's do it in a separate PR.

Reply:
Alright!